src/audio/rtnr/rtnr.c

```diff
@@ -407,12 +407,16 @@ static int rtnr_get_comp_data(struct comp_data *cd, struct sof_ipc_ctrl_data *cd
 	if (size > max_data_size || size < 0)
 		return -EINVAL;
 
-	ret = memcpy_s(cdata->data->data,
-				max_data_size,
-				config,
-				size);
-	if (ret)
-		return ret;
+	if (size > 0) {
```

Comment:
can't this go on the else branch above?

Reply:
Calling memcpy_s with size equals 0 will cause problems on my side. Hence I only do the copy when the data size is valid.